example/src/pages/index.tsx

```diff
@@ -52,14 +57,16 @@ const IndexPage = () => {
           case QueryParams.FeeStrategy:
             setFeeSelected(value);
             break;
-          default:
-            customConfig[key] = value;
+          case InternalParams.Rate:
+            setRate(+value);
```

Comment:
What is the '+' in front of the `value` is for?

Reply:
convert string (from query) in number